src/test/K6/src/tests/platform/authorization/delegations/delegationsv2.js

```diff
@@ -0,0 +1,379 @@
+/*
+  Test data required: deployed app (reference app: ttd/apps-test)
```

Comment:
Should the description of these files say something about which test scenarios the file is to contain?

What is the difference between delegations.js and delegationsv2.js?
From what I can tell by the test method names this is direct delegations to a user or an organization.

Same goes for inheritance.js and inheritancev2.js. Why do we need the two versions and which scenarios are to be collected in it?
The name should suggest only scenarios for inheritance of rights through keyroles or from main unit, but the file also contains methods:
directDelegationFromOrgToUser();
directDelegationFromOrgToOrg();

Reply:
I'm keeping two versions of the same test files to help transition to the new version. I'm updating the pipeline and release settings, and using the old v1 as fallbacks if things don't work. We could merge delegationsv2.js and inheritancev2.js into one file, because they use the same test data and test the same endpoints.